mypy/messages.py

```diff
@@ -2944,7 +2944,7 @@ def make_inferred_type_note(
 def format_key_list(keys: list[str], *, short: bool = False) -> str:
     formatted_keys = [f'"{key}"' for key in keys]
     td = "" if short else "TypedDict "
-    if len(keys) == 0:
+    if not keys:
```

Comment:
would prefer to keep this one as-is, as it retains symmetry with the `elif` immediately below